lambda/aws-qa-appsync-opensearch/question_answering/src/qa_agent/image_qa.py

```diff
@@ -269,18 +260,9 @@ def generate_vision_answer_sagemaker(_qa_llm,input_params,decoded_question,statu
     
     return status_variables
 
-def generate_vision_answer_bedrock(bedrock_client,base64_images,model_id,decoded_question):
-    system_prompt=""
-    # use system prompt for fine tuning the performamce
-    # system_prompt= """
-    # You have perfect vision and pay great attention to detail which
-    # makes you an expert at answering architecture diagram question. 
-    # Answer question in <question></question> tags. Before answer, 
-    # think step by step in <thinking> tags and analyze every part of the diagram.
-    # """
-    #Create a prompt with the question
-    prompt =f"<question>{decoded_question}</question>. Answer must be a numbered list in a small paragraph inside <answer></answer> tag."
-
+def generate_vision_answer_bedrock(bedrock_client,base64_images,model_id,
+                                   decoded_question,status_variables,streaming):
+    
     claude_config = {
         'max_tokens': 1000, 
         'temperature': 0, 
```

Comment:
do you want to pass the awsjson parameters here instead of harcoded values ?

Reply:
I am working on another PR which will use BedrockChat , that should cover the aws json parameters.